src/gui/optionsdlg.ui

```diff
@@ -3046,6 +3046,31 @@ Use ';' to split multiple entries. Can use wildcard '*'.</string>
                  </layout>
                 </widget>
                </item>
+               <item>
+                <widget class="QGroupBox" name="groupAltWebUI">
+                 <property name="title">
+                  <string>Use alternative Web UI</string>
+                 </property>
+                 <property name="checkable">
+                  <bool>true</bool>
+                 </property>
+                 <property name="checked">
+                  <bool>false</bool>
+                 </property>
+                 <layout class="QGridLayout" name="gridLayout_8">
+                  <item row="0" column="0">
+                   <widget class="QLabel" name="labelWebUIRootFolder">
+                    <property name="text">
+                     <string>Files location:</string>
```

Comment:
`Folder location:`

Reply:
Files location is path of its parent folder. But folder location is the place where the folder is located (i.e.its parent folder path). So current wording is correct.